packages/connect/src/data/defaultFeeLevels.ts

```diff
@@ -0,0 +1,130 @@
+import { FeeLevel, FeeInfo } from '../types';
+
+// this is workaround for the lack of information from 'trezor-common'
+// we need to declare what does "high/normal/low" mean in block time (eg: normal BTC = 6 blocks = ~1 hour)
+// coins other than BTC usually got 2 levels maximum (high/low) and we should consider to remove other levels from 'trezor-common'
+const BLOCKS_FOR_FEE_LEVEL: Record<string, Record<string, number>> = {
```

Comment:
not part of  this PR, just a general thought - this information (fees) could become part of the new config we are planning together with blockchain-link information and others that we don't want to use from trezor-common anymore.